habitat-lab/habitat/isaacsim/urdf_conversion.py

```diff
@@ -0,0 +1,48 @@
+"""This module converts urdf files to usd.""" 
+# TODO: first setup mamba activate isaaclab2?
+
+import argparse
+from omni.isaac.lab.app import AppLauncher
+
+def convert_urdf_to_usd(source_urdf_filepath: str, usd_folder_path: str) -> None:
+    """
+    This function converts a .urdf file into a .usd file. NOTE: Please be aware of 
+    pathing issues and references to other .glb files within the urdf file.
+    
+    :param source_urdf_filepath: The absolute string path of the urdf file
+    :param usd_folder_path: The absolute directory of the desired directory location of .usd file. The output will generatate a Props-> instanceable_meshes.usd, .asset_hash, config.yaml, and the desired .usd.
+
+    """
+
+    # create argparser
+    parser = argparse.ArgumentParser(description="Create an empty Issac Sim stage.")
```

Comment:
Let's not create AppLauncher inside the function, because we'll likely call this function several times inside a larger application in the future. You can create the AppLauncher in your `if __name__ == "__main__":` block so that the script still works standalone.

Reply:
Applauncher class moved.